docs/app/interfaces/IPlunk.ts

```diff
@@ -14,6 +14,11 @@ export interface IPlunk {
         alias: string;
         source: string;
     }[];
+    routes?: {
+        path: string;
+        pathMatch?: string;
+        component: string
+    }[];
```

Comment:
Hello Pearse. Please don't add this, it's only useful for your one example. All you need is to allow a custom module template. Can you do that instead? And check things like this before spending time on them?